editors/emacs/prettier-js.el

```diff
@@ -219,4 +219,13 @@ function."
      (delete-file bufferfile)
      (delete-file outputfile)))
 
+;;;###autoload
+(define-minor-mode prettier-mode
+  "Runs prettier on file save when this mode is turned on"
+  :lighter " prettier"
+  :global t
+  (if prettier-mode
+      (add-hook 'before-save-hook 'prettier-before-save)
```

Comment:
I made a pull request https://github.com/prettier/prettier/pull/1623 with this change, I guess we could move this discussion there.

Thanks!